### PR TITLE
NFT tutorial

### PR DESCRIPTION
**Pull request by jmcanterafonseca-iota**
# Description of change

NFT tutorial

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Tutorial

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by jmcanterafonseca-iota** [comment]
@lucas-tortora you can complement it with a PR as usual :)

File: documentation/docs/tutorials/nft-transactions/03-mint-new-nft.md [hunk 1]
```diff
@@ -11,19 +12,34 @@ keywords:
...
+usually also immutable metadata that links the NFT with a Digital Asset (image, video, etc.). Non-immutable aspects of
+the NFT can only be changed by the owner of the NFT by unlocking its unspent Output in a transaction, and generating a
+new Output (conserving the NFT ID). A transaction can also be used to transfer the NFT to another owner. As it happens
+with other types of Outputs, NFT Outputs must also cover its storage costs with protocol-defined tokens (`SMR`).
 
 In order to mint a new NFT it is needed:
 
-* A not spent [Basic Output]() that holds enough funds for the minimal storage deposit needed for the NFT Output. In the testnet you can provision funds through the Faucet, as we explained in our [previous tutorial](https://wiki.iota.org/shimmer/iotajs/tutorials/value-transactions/request-funds-from-the-faucet/).
+* A not spent [Basic Output](https://wiki.iota.org/shimmer/tips/tips/TIP-0018/#basic-output) that holds enough funds for
+  the minimal storage deposit needed for the NFT Output. In the testnet you can provision funds through the Faucet, as
```
Comment (jmcanterafonseca-iota) [inline documentation/docs/tutorials/nft-transactions/03-mint-new-nft.md]: Testnet

File: documentation/docs/tutorials/nft-transactions/04-nft-transaction.md [hunk 2]
```diff
@@ -34,15 +45,17 @@ const nftOwnerPrivateKey = "0x22f6...";
 const nftBuyerAddr = "0x57d3...";
 ```
 
-## Query NFT Output
+## Query For the NFT Output
 
-You will first need to find the NFT Output of your NFT. The easiest way to do so is through a query to the [indexation plugin](https://wiki.iota.org/shimmer/inx-indexer/welcome/) by NFT ID. Observe that you need to obtain the full Output details as you need to use them as input for the transaction.
+You will first need to re the NFT Output of your NFT. The easiest way to do so is through a query to
```
Comment (jmcanterafonseca-iota) [inline documentation/docs/tutorials/nft-transactions/04-nft-transaction.md]: query the NFT Output

File: documentation/docs/tutorials/nft-transactions/03-mint-new-nft.md (hunk 1 above)
Comment (lucas-tortora) [reply documentation/docs/tutorials/nft-transactions/03-mint-new-nft.md]:
```suggestion
  the minimal storage deposit needed for the NFT Output. In the Testnet you can provision funds through the Faucet, as
```

File: documentation/docs/tutorials/nft-transactions/04-nft-transaction.md (hunk 2 above)
Comment (lucas-tortora) [reply documentation/docs/tutorials/nft-transactions/04-nft-transaction.md:50]:
```suggestion
You will first need to query for the NFT Output of your NFT. The easiest way to do so is through a query to
```

**Review by lucas-tortora — APPROVED** [review]
LGTM :)

**Pull request merged**